packages/contracts/src/networks.ts

```diff
@@ -6,11 +6,16 @@ export type SupportedNetwork = {
     apiURL: string
     browserURL: string
     envKey: string
+    blockExplorer: BlockExplorer
   }
   currency: string
   dripSize: string
   requiredEnvVariables: Array<string>
   networkType: NetworkType
+  dripVersion: number
+  decimals: number
+  queryFilterBlockLimit: number
+  legacyTx: boolean
```

Comment:
Added a few new fields to the networks info.

The `dripVersion` field makes it easy for us to update the drip sizes on different networks. We just have to update the dripSize, bump the dripVersion, and then we can run the deploy script and it will archive any previous Drippie configurations and replace them with the new version. 

The `decimals` field is used whenever we need to handle currency conversions (mainly on the website).

The `queryFilterBlockLimit` is also just used by the website; it's for cases where networks limit the block range you can use when querying events.

The `legacyTx` field allows us to support networks that do not use eip-1559 transactions where we must use a `--legacy` flag when calling into forge to collect the transactions.